configuration/configuration.go

```diff
@@ -51,7 +51,16 @@ func Load(configFile string) (*Configuration, error) {
 	var c Configuration
 	c.root = pwd
 
-	fi, err := os.Stat(configFile)
+	f, err := os.Open(configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return &c, nil
+		}
+		return nil, err
+	}
+	defer f.Close()
+
+	fi, err := f.Stat()
```

Comment:
This `.Stat` is probably not needed anymore since you cannot `os.Open` on a directory.